add swagger dependency

syevtushok opened the pull request:
add jackson-dataformat-csv
add entities
add some repository implementations

syevtushok committed:
add swagger dependency
add jackson-dataformat-csv
add entities
add some repository implementations

syevtushok committed:
troubles with travis

boroda4436 commented on src/main/java/mate/academy/review/analysis/entity/Product.java:
```diff
@@ -0,0 +1,24 @@
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String productId;
```
@ggazila what is the difference between id and productId? Is the productId unique?

boroda4436 commented on src/main/java/mate/academy/review/analysis/repository/ProfileRepository.java:
```diff
@@ -0,0 +1,4 @@
+package mate.academy.review.analysis.repository;
+
+public interface ProfileRepository {
```
@ggazila consider adding two more repositories for product and review

syevtushok committed:
change structure of models
add FileReader(get from filesystem or download)
add csv parser
add implementation to write reviews into database
